src/features/MainPage/index.js

```diff
@@ -82,14 +63,15 @@ function MainPage() {
           <Pagination
             currentPage={currentPage}
             minPageLimit={1}
-            maxPageLimit={100}
+            maxPageLimit={400}
```

Comment:
Kamil told something about 500 pages in Paginator

Reply:
I know but there's a problem with the limit of pages we can download. According to [this](https://www.themoviedb.org/talk/62ee92a99369a2007dc23ffa#:~:text=There%20will%20always%20be%20up,a%20maximum%20of%20500%20pages.) we can only download maximum 500 pages. Each page means 20 elements, so if we want to show on page 24 elements for popular people sites we need to download 600 pages from Api. That's why I changed maxPageLImit to 400